### PR TITLE
feat: Generate Swagger docs automatically using semantic release [DEV-2773]

### PR DESCRIPTION
**Pull request by abdulla-ashurov**
- [x] Implement the necessary changes to the codebase to enable automatic generation of Swagger docs.
  - [x] Verify that the Swagger docs are correctly generated based on the annotated code.
- [ ] Configure the semantic release process to trigger the Swagger documentation generation automatically.

### CODE REVIEW

**Comment by ankurdotb** [comment]
Task linked: [DEV-2773 Generate Swagger docs automatically using semantic release](https://app.clickup.com/t/6600954/DEV-2773)

File: src/app.ts [hunk 1]
```diff
@@ -14,14 +14,49 @@ import { Connection } from './database/connection/connection.js'
...
-
 import * as dotenv from 'dotenv'
 dotenv.config()
 
 import path from 'path'
 import { LogToWebHook } from './middleware/hook.js'
 import { Middleware } from './middleware/middleware.js'
+import swaggerJsdoc from 'swagger-jsdoc'
+
+const options = {
```
Comment (DaevMithran) [inline src/app.ts:25]: Better to move this into a separate file in /static folder

File: src/controllers/customer.ts [hunk 2]
```diff
@@ -23,6 +58,7 @@ export class CustomerController {
         }
     }
 
+    // This method never use.
     public async update(request: Request, response: Response) {
```
Comment (DaevMithran) [inline src/controllers/customer.ts:62]: Let's remove it then, not safe to let a customer update the details

File: src/controllers/customer.ts [hunk 3]
```diff
@@ -5,6 +5,41 @@ import { LogToHelper } from '../middleware/auth/logto.js'
 
 export class CustomerController {
```
Comment (DaevMithran) [inline src/controllers/customer.ts:6]: There are there terms used, account, customer, client. Better to change everything to account

File: src/controllers/issuer.ts [hunk 4]
```diff
@@ -64,6 +98,46 @@ export class IssuerController {
...
+   *         required: true
+   *     responses:
+   *       200:
+   *         description: The request was successful.
+   *         content:
+   *           application/json:
+   *             schema: 
+   *               $ref: '#/components/schemas/KeyResult'
+   *       400:
+   *         description: A problem with the input fields has occurred. Additional state information plus metadata may be available in the response body.
```
Comment (DaevMithran) [inline src/controllers/issuer.ts:123]: Better to create a $ref for status code's 400 and 500, similar to 401. So that we can reuse it

**Review by DaevMithran — CHANGES_REQUESTED** [review]

File: src/swagger.cjs
Comment (DaevMithran) [inline src/swagger.cjs:1]: Can u move this file into /static

**Review by DaevMithran — CHANGES_REQUESTED** [review]

File: src/controllers/customer.ts (hunk 2 above)
Comment (abdulla-ashurov) [reply src/controllers/customer.ts:62]: Removed! ✅

File: src/controllers/customer.ts (hunk 3 above)
Comment (abdulla-ashurov) [reply src/controllers/customer.ts:6]: Done! ✅

File: src/swagger.cjs
Comment (abdulla-ashurov) [reply src/swagger.cjs:1]: Done! ✅

**Review by DaevMithran — APPROVED** [review]

**Review by ankurdotb — CHANGES_REQUESTED** [review]
Swagger file generated has issues. Please check ClickUp

**Comment by DaevMithran** [comment]
@abdulla-ashurov @ankurdotb It was my mistake. Fixed it

**Review by ankurdotb — APPROVED** [review]

**Pull request merged**